skimage/transform/radon_transform.py

```diff
@@ -62,6 +67,17 @@ def radon(image, theta=None, circle=True):
     if theta is None:
         theta = np.arange(180)
 
+    if preserve_range is None and np.issubdtype(image.dtype, np.integer):
+        warn('Image dtype is not float. By default radon will assume'
+             'you want to preserve the range of your image'
+             '(preserve_range=True). In scikit-image 0.18 this behavior will'
+             'change to preserve_range=False. To avoid this warning,'
+             'explictiely specify the preserve_range parameter.',
+             DeprecationWarning)
```

Comment:
I don't like `DeprecationWarning`, it tends to get hidden if it isn't called directly by `__main__`, and users won't get the warning. Let it be a standard warning, or `FutureWarning`.

```suggestion
             stacklevel=2)
```

For us, Deperecations will actually break things for users in 2 versions (1 year).

Reply:
Done